docs/mobile-apps/automated-testing/espresso-xcuitest/real-devices.md

```diff
@@ -179,10 +212,10 @@ From here, head to the full list of commands and options at [Sauce Runner for Re
 
 As an alternative to configuring your Espresso and XCUITest RDC tests using the [command line interface](/dev/cli/espresso-xcuitest.md), you can create and run a YAML configuration file.
 
-Add the [`config` command](dev/cli/espresso-xcuitest.md) to your test script. The config command only accepts two parameters: `--path <path to config.yml>` and `--apikey <apikey>`. Here's an example:
+Add the [`config` command](dev/cli/espresso-xcuitest.md) to your test script. The config command only accepts two parameters: `--path <your path to config.yml>` and `--accessKey <your_accessKey>`. Here's an example:
```

Comment:
@kimsaucelabs 

You also need to have the `--username $SAUCE_USERNAME ` in the command, this will not work